ADRC_X_r: 2
ADRC_X_h: 0.01
ADRC_X_N: 15
ADRC_X_beta_01: 300
ADRC_X_beta_02: 200
ADRC_X_beta_03: 20
ADRC_X_b0: 200
ADRC_X_k_0: 0.002
ADRC_X_k_1: 1.6
ADRC_X_k_2: 0.02
ADRC_X_N1: 5
ADRC_X_C: 5
ADRC_X_alpha1: 0.8
ADRC_X_alpha2: 1.5
ADRC_X_zeta: 50
ADRC_Y_r: 2
ADRC_Y_h: 0.01
ADRC_Y_N: 15
ADRC_Y_beta_01: 300
ADRC_Y_beta_02: 200
ADRC_Y_beta_03: 20
ADRC_Y_b0: 200
ADRC_Y_k_0: 0.002
ADRC_Y_k_1: 1.6
ADRC_Y_k_2: 0.02
ADRC_Y_N1: 5
ADRC_Y_C: 5
ADRC_Y_alpha1: 0.8
ADRC_Y_alpha2: 1.5
ADRC_Y_zeta: 50
ADRC_Z_r: 30
ADRC_Z_h: 0.01
ADRC_Z_N: 60
ADRC_Z_beta_01: 200
ADRC_Z_beta_02: 40
ADRC_Z_beta_03: 20
ADRC_Z_b0: 260
ADRC_Z_k_0: 0.002
ADRC_Z_k_1: 3
ADRC_Z_k_2: 0.12
ADRC_Z_N1: 5
ADRC_Z_C: 5
ADRC_Z_alpha1: 0.8
ADRC_Z_alpha2: 1.5
ADRC_Z_zeta: 50
